ansible/system.yml

```diff
@@ -48,7 +55,7 @@
   become: yes
   roles:
     - role: geerlingguy.docker
-      when: (install_splash is defined and install_splash) or (install_opencast_importer is defined and install_opencast_importer)
+      when: ((install_splash is defined and install_splash) or (install_opencast_importer is defined and install_opencast_importer)) and  edu_version is version('7.0.0','<')
```

Comment:
I expect splash is not necessary from 7.0.0? Or why do you exclude the docker-installation here?

Nevertheless, opencast-importer also has to be installable from 7.0.0